README.rst

```diff
@@ -3,7 +3,11 @@ sqlalchemy-vertica-python
 
 Vertica dialect for sqlalchemy. Forked from the `Vertica ODBC dialect <https://pypi.python.org/pypi/vertica-sqlalchemy>`_.
 
-This module implements a Vertica dialect for SQLAlchemy using `vertica-python <https://github.com/uber/vertica-python>`_. Engine creation: 
+This module implements a Vertica dialect for SQLAlchemy using `vertica-python <https://github.com/uber/vertica-python>`_
```

Comment:
Should this point at the official vertica repo now?

Reply:
Done